src/content/docs/dropins/payment-services/installation.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Payment Services installation
+description: Learn how to install the Payment Services drop-in component on your site.
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+
+Before you can use the Payment Services component on your storefront, you must [configure the Services Connector](https://experienceleague.adobe.com/en/docs/commerce-learn/tutorials/admin/adobe-commerce-services/configure-adobe-commerce-services-connector) and [onboard](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/payment-services/get-started/onboard) the Payment Services extension in the Adobe Commerce Admin.
+
+:::note
+The [Payment Services extension](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/payment-services/release-notes#v2100) supports drop-in component capabilities from versions 2.10.0 and newer.
```

Comment:
I would phrase it the other way around, i.e., from the POV of the drop-ins, not from the POV of the Luma extension. Something like: the Payment Services drop-in requires the Payment Services extension 2.10 or higher.